src/examples/Country.ts

```diff
@@ -0,0 +1,8 @@
+interface Country extends Context {
+  type: 'fdc3.country',
+  name?: string;
```

Comment:
The optional name property is already inherited from the base context type. You can include it for completeness if you like, but it is redundant.

Reply:
for a country, you will need a name, so the ? should be removed

Reply:
Agreed, the country is not much use without a name.